### ISSUE TITLE
Our Research Interests

### ISSUE THREAD

**Issue by NatalieZelenka** [very-good-science/our-handbook-template#11]
Info: Depends on group size and structure. Link to other sections where possible. Metadata?
Examples: Small group, medium group, big group.

**Comment by lauren-cadwallader** [comment]
I'm going to have a stab at this one today.

**Comment by ekuw** [comment]
Would it make sense to link here from "Our Team" profiles to give team members more room to talk about their research?

**Comment by lauren-cadwallader** [comment]
Yes I think either from this page or the projects page. I will put in a link to the Our Team page from Research Interests.
